app/resources/ephemera_fields/ephemera_field_decorator.rb

```diff
@@ -21,6 +21,50 @@ def favorite_terms
     wayfinder.decorated_favorite_terms
   end
 
+  def rarely_used_terms
+    wayfinder.decorated_rarely_used_terms
+  end
+
+  # Returns categories or terms with rarely used terms at the end and favorite
+  # terms at the beginning.
+  def sorted_terms_or_categories
+    return sorted_categories if attribute_name == "subject"
+    non_favorite_terms = vocabulary.terms.select { |x| !favorite_term_ids.include?(x.id) && !rarely_used_term_ids.include?(x.id) }
+    favorite_terms + non_favorite_terms + rarely_used_terms
+  end
+
+  # Returns categories where favorite terms appear at the top and rarely used
```

Comment:
So favorite terms appear in both places. It might help to add this to the comment.